src/Utilities/hooks/useTableTools/testHelpers.js

```diff
@@ -1,14 +1,18 @@
-import { render } from '@testing-library/react';
+import { render, act } from '@testing-library/react';
 import { queryByText } from '@testing-library/dom';
 import userEvent from '@testing-library/user-event';
 
 const toggleFilterDropDown = (container) => {
   const filterToggle = container.querySelector('button.pf-c-dropdown__toggle');
-  userEvent.click(filterToggle);
+  act(() => {
+    userEvent.click(filterToggle);
+  });
```

Comment:
nit-pick:
```suggestion
    await userEvent.click(filterToggle);
```

would also require marking the function as async

Reply:
These helpers need to be completely rewritten. I tried, but I think it'll be done in a following PR.